setup.py

```diff
@@ -51,12 +51,20 @@
 
 
 def here(*paths):
+    return os.path.relpath(
+        abshere(*paths),
+        os.path.abspath(os.path.dirname(__file__)),
+    )
+
+
+def abshere(*paths):
     return os.path.abspath(os.path.join(os.path.dirname(__file__), *paths))
```

Comment:
Err, wouldn't it make more sense to have `abshere` be implemented in terms of `here`?